packages/slice-machine/src/features/onboarding/content.tsx

```diff
@@ -6,94 +6,118 @@ import type {
   OnboardingStepType,
 } from "@/features/onboarding/types";
 
-export const onboardingSteps: OnboardingStep[] = [
+export const onboardingSteps = [
   {
-    id: "addSlice",
-    title: "Add Slices",
-    description: "Add slices to your page type.",
+    id: "createPageType",
+    title: "Create a Page Type",
+    description: "Build the structure for your page.",
   },
   {
     id: "codePage",
-    title: "Code your page",
-    description: "Build your page's structure",
+    title: "Code Your Page Type",
+    description: "Prepare your code to fetch content.",
   },
   {
-    id: "pushModels",
-    title: "Push Your Page",
-    description: "Push to your Page Builder.",
+    id: "createSlice",
+    title: "Create a Slice",
+    description: "Build a reusable website section.",
+  },
+  {
+    id: "reviewAndPush",
+    title: "Review and Push Changes",
+    description: "Enable editors to create content.",
   },
   {
     id: "createContent",
-    title: "Create content",
-    description: "Create engaging content.",
+    title: "Create Content in the Page Builder",
+    description: "Publish a page to the Prismic API.",
   },
   {
     id: "renderPage",
     title: "Render your page",
-    description: "Render your page on localhost.",
+    description: "View your page in the browser.",
   },
-];
+] as const satisfies OnboardingStep[];
 
-type GetOnboardingStepsContentProps = {
-  repositoryUrl: string;
+type OnboardingStepIndexNumeralMap = {
+  [K in Exclude<keyof typeof onboardingSteps, keyof unknown[]>]: string;
 };
 
-export const getOnboardingStepsContent = ({
-  repositoryUrl,
-}: GetOnboardingStepsContentProps): Record<
+export const stepIndexNumeral: Record<number, string> = {
+  0: "one",
+  1: "two",
+  2: "three",
+  3: "four",
+  4: "five",
+  5: "six",
+} satisfies OnboardingStepIndexNumeralMap;
```

Comment:
This isn't really supposed to be related to the steps, the type is only there to prevent you from adding a step and forgetting to add a label. If the numeral label for the index lives next to the step, you have to update it if you change the order of the steps.

Reply:
But this has to go, unfortunately, `satisfies` doesn't work with next 12 and I can't do any of this type checking 😔

Reply:
Since I cannot type check it correctly right now, changed to a simple unchecked array.

Reply:
What I meant by "not related" is that they are simply a map of number (index) to label. The only reason I added a `satisfies` was to make sure we have enough labels for every step, so that I don't have to create labels up to 20 (arbitrary number) just to be safe.

Anyway, I changed it 😅

Reply:
Final update: Asked benjamin if we could display a number. Changed it to a number.